Source/RapyutaSimulationPlugins/Private/Tools/ROS2Spawnable.cpp

```diff
@@ -8,7 +8,7 @@
 void UROS2Spawnable::InitializeParameters(const FROSSpawnEntity_Request& InRequest)
 {
     SetName(InRequest.state_name);
-    SetNamespace(InRequest.robot_namespace);
+    SetNamespace(InRequest.robot_namespace.Replace(TEXT("/"),TEXT("")));
```

Comment:
we can create utility function for sanity check for this in the future, I think. 
Because of ros convention, there maybe multiple place to check this.